ckanext/xloader/plugin.py

```diff
@@ -23,7 +25,7 @@ def config_declarations(cls):
 class xloaderPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IConfigurable)
-    plugins.implements(plugins.IResourceUrlChange)
+    plugins.implements(plugins.IDomainObjectModification)
```

Comment:
Does this exist in ckan 2.9?

Reply:
I believe so. I think it is even in CKAN 2.8 but I will double check to be sure that it wasn't something that we just backported into our 2.8/2.9 forks

Reply:
@duttonw yes, can confirm this exists in CKAN 2.8+